packages/webpack-graph-ql/src/lib/schema.ts

```diff
@@ -0,0 +1,140 @@
+import resolvers from './resolvers';
+const schema = `
+type Query {
+  stats: Stats
+}
+
+type Stats {
+  assets: [Asset]
+  chunks: [Chunk]
+  modules: [Module]
+  errors: [Error]
+  warnings: [Warning]
+  hash: String
+  version: String
+  time: Int
+}
+
+type Asset {
+  name: String
+  size: Int
+  chunkNames: [String]
+}
+
+type Chunk {
+  id: String
+  names: [String]
+  modules: [Module]
+  origins: [Origin]
+}
+
+type Module {
```

Comment:
It'd be really nice to generate the typedefs.

Reply:
How would I do that? I'm not the best at graphQL

Reply:
Something kinda like this was my vision https://github.com/module-federation/universe/pull/475